### PR TITLE
Let ExecutorService be jackson serializable

### PR DESCRIPTION
**Pull request by thespags**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -63,7 +63,7 @@
         <!-- NOTE: Must match what's available in governator.version -->
         <javax.el>3.0.0</javax.el>
         <javax.servlet-api.version>3.1.0</javax.servlet-api.version>
-        <javax.ws.rs-api.version>2.0.1</javax.ws.rs-api.version>
+        <javax.ws.rs-api.version>2.1</javax.ws.rs-api.version>
```
Comment (timkral) [inline pom.xml:66]: Why do you need this update?

**Comment by thespags** [comment]
@timkral  whats the status of this pr for adding serialization?

**Comment by timkral** [comment]
@thespags , I was curious why you did the JAX-RS update.

**Comment by thespags** [comment]
I was upgrading it in drunkr so that Response becomes autocloseable.(They still haven't made The request autocloseable with a lot of discussion about why they can't do it yet).
I wanted to make the version aligned in both projects.

**Comment by timkral** [comment]
Ok, sounds good.

**Review by timkral — APPROVED** [review]

**Pull request merged**